### DISCUSSION TITLE
Does Twind support TailwindCSS v.3 colors?

### DISCUSSION THREAD

**Discussion by yabdab** [tw-in-js/twind#450] [Q&A]
I am trying to use Tailwindcss v.3 colors, but they do not seem to be included?

Does Twind still use v.2 or is there a way to get all the default colors in v.3?

Thanks.

**Comment by yabdab** [comment]
Is twind actively developed any more? No replies to this question in over 2 months.

**Reply by benebsiny** [reply]
It seems that this project isn't developed actively. There's no critical update since January.
